docs/cli-usage.md

```diff
@@ -0,0 +1,17 @@
+# CLI Usage
+
+The following usage documentation is for the [Semgrep CLI](https://github.com/returntocorp/semgrep).
+
+[TOC]
```

Comment:
TOC is probably not needed here? This doc is super short.

Reply:
I've gotten into the practice of adding them. TOC provides anchor links too, should that be nice. Happy to remove though.

Reply:
Ah yes, probably useful for the anchors alone. 👍